prow/config/config.go

```diff
@@ -381,6 +381,12 @@ type Plank struct {
 	// DefaultDecorationConfig are defaults for shared fields for ProwJobs
 	// that request to have their PodSpecs decorated
 	DefaultDecorationConfig *prowapi.DecorationConfig `json:"default_decoration_config,omitempty"`
+
+	// DecorationConfigByRepo holds the default decoration config for specific repositories.
+	// This config will be used only on Presubmits and Postsubmits and the missing fields will
+	// be merged with the DefaultDecorationConfig.
+	DecorationConfigByRepo map[string]*prowapi.DecorationConfig `json:"decoration_config_by_repo,omitempty"`
```

Comment:
nit: these are still defaults, and if we intend to migrate away from the old option we could do well to keep the names similar, like `default_decoration_configs`

Reply:
addressed